media/editor/dataInspector.tsx

```diff
@@ -112,6 +113,10 @@ const InspectorContents: React.FC<{
 					<dd>{target.length < minBytes ? <span className={endOfFileCls}>End of File</span> : convert(dv, le)}</dd>
 				</React.Fragment>
 			)}
+			<React.Fragment key={"Byte Count"}>
+				<dt>{"Byte Count"}</dt>
+				<dd>{target.length < 1 ? <span className={endOfFileCls}>End of File</span> : selectionCount.toString()}</dd>
+			</React.Fragment>
```

Comment:
I think we can remove this now since we're using the status bar

Reply:
ah yep, forgot about this